README.md

```diff
@@ -71,5 +60,3 @@ Give a ⭐️ if you like this project!
 ## 📝 License
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link of MIT.md is broken, kindly fix it by changing the path to `[MIT](./MIT.md)` instead of `[MIT](./LICENSE)` 💯
- [ ] Also, I noticed that you forgot to customize your `MIT.md` file, kindly add your name and customize it 👌

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

![image](https://user-images.githubusercontent.com/45231276/191185249-5efc9289-26e9-4ceb-80f0-e5bd0a3f1755.png)